docs/DEVELOPER_GUIDE.md

```diff
@@ -1 +1,179 @@
-todo
\ No newline at end of file
+# Development
+
+This doc explains how to setup a development environment so you can get started
+[contributing](../CONTRIBUTING.md)
+to `kfserving`. Also take a look at:
+
+- [How to add and run tests](../test/README.md)
+- [Iterating](#iterating)
```

Comment:
I think this might be a bad link? Capital I?

Reply:
hmm works fine for me